data/RI/incentives.json

```diff
@@ -245,6 +245,32 @@
       "es": "Un reembolso adicional de $1,500 por un vehículo eléctrico usado para los calificadores de LIHEAP. Por orden de llegada."
     }
   },
+  {
+    "id": "RI-13",
+    "authority_type": "state",
+    "authority": "ri-oer",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "ebike",
+    "program": "ri_erikaNiedowskiMemorialEBikeRebateProgram",
+    "amount": {
+      "type": "percent",
+      "number": 0.3,
+      "maximum": 350
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "low_income": "default",
+    "short_description": {
+      "en": "30% up to $350 back on the final purchase of an e-bike or e-cargo bike. For income-qualified residents, 75% up to $750 is available.",
```

Comment:
In this case we'd rather have the income-qualified version expressed as a separate incentive, and have an `exclusion` relationship between them, expressing "if you claim the low-income version, you can't also claim the non-restricted version".

You can look at VT-55 and VT-56 as an example, but the steps are basically:

- Add the low-income version as its own incentive. (I think you can say it's under the same program.)

- Add a separate file `data/RI/incentive_relationships.json`:
  ```json
  {
    "exclusions": {
      "RI-XX": ["RI-YY"]
    }
  }
  ```
  The low-income incentive's ID should be the key in the map, which expresses that it's the one that "blocks out" the other.

- Add code to read that file in `state_incentive_relationships.ts`.

And ideally it should be two separate rows in the spreadsheet too, to ease eventually using the script to do the translation.

Reply:
updated!